projects/aca-content/src/lib/components/sidenav/navigation-menu/navigation-menu.component.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2005 - 2021 Alfresco Software, Ltd. All rights reserved.
+ *
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
+ */
+
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { ContentApiService } from '@alfresco/aca-shared';
+
+@Component({
+  selector: 'app-navigation-menu',
+  templateUrl: './navigation-menu.component.html',
+  encapsulation: ViewEncapsulation.None
+})
+export class NavigationMenuComponent implements OnInit {
+  displayName = 'Unknown user';
```

Comment:
I think it should be provided in translation files

Reply:
will make empty and add "ngIf" to the html template